contracts/ColonyStorage.sol

```diff
@@ -185,6 +193,16 @@ contract ColonyStorage is DSAuth, DSMath {
     _;
   }
 
+  modifier recovery() {
+    require(recoveryMode, "not-in-recovery-mode");
+    _;
+  }
+
+  modifier normal() {
+    require(!recoveryMode, "in-recovery-mode");
+    _;
+  }
```

Comment:
I should have probably remembered to mention this back when you commented on how you've agreed to do it but it just dawned on me this type of logic is already available in the dappsys lib we might be able to reuse https://github.com/dapphub/dappsys-monolithic/blob/002389d43cf54e8f0b919fee1fc364b20ebdf626/stop.sol#L23